suites/reef/rgw/tier-1_rgw_multisite.yaml

```diff
@@ -312,42 +312,6 @@ tests:
       module: sanity_rgw_multisite.py
       name: Manual Resharding tests on Primary cluster
       polarion-id: CEPH-83574734
-  - test:
```

Comment:
Please add this tc and next tc in [
reef/rgw/tier-1_rgw_ms_ecpool_regression_test](https://github.com/red-hat-storage/cephci/blob/master/suites/reef/rgw/tier-1_rgw_ms_ecpool_regression_test.yaml#L404)

Reply:
@ckulal , Added the 2 listing tests to suite reef/rgw/tier-1_rgw_ms_ecpool_regression_test